### ISSUE TITLE
Time conversion for PHITS

### ISSUE THREAD

**Issue by dddijulio** [mctools/mcpl#53]
The PHITS unit for time is ns, and the conversion during phits2ssw should be updated to 1E-6 instead of 1E6

**Comment by tkittel** [comment]
Thanks! So I guess you meant phits2mcpl. It seems we have a similar, but reverse, issue in mcpl2phits.

So the two lines in question have buggy code but correct comments:

```c++
    mcpl_particle->time = p->time * 1.0e6;//nanoseconds (PHITS) to milliseconds (MCPL)
    dumpdata[9] = mcpl_p->time * 1.0e-6;//ms->ns
```
Which should of course be changed to:
```c++
    mcpl_particle->time = p->time * 1.0e-6;//nanoseconds (PHITS) to milliseconds (MCPL)
    dumpdata[9] = mcpl_p->time * 1.0e6;//ms->ns
```
I'll see if I can find time to fix this and make a new release soonish. In the meantime, it would be great if you can verify that there are no other fixes needed.

**Comment by dddijulio** [comment]
sorry, I did mean phits2mcpl!

**Comment by tkittel** [comment]
Fixed in release 1.3.2, closing here :-)
